engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -662,7 +609,6 @@ private void renderChunk(RenderableChunk chunk, ChunkMesh.RenderPhase phase, Cam
     public void dispose() {
         renderableWorld.dispose();
         worldProvider.dispose();
-        CoreRegistry.get(AudioManager.class).stopAllSounds();
```

Comment:
Moved to StateIngame.dispose() method. 

Here I've also been wondering if it would be appropriate to dispose of the BackdropProvider and the BackdropRenderer. Neither have a dispose() method at this stage, but perhaps they should? And is it appropriate to dispose of them here? They get instantiated (indirectly) in StateLoading.step() and perhaps they should be disposed of in StateIngame.dispose()? 

And the disposal of WorldProvider should perhaps also happen in StateIngame.dispose()? It just doesn't seem right that it gets disposed here.